docker-compose.yaml

```diff
@@ -15,23 +25,34 @@ services:
       - SERVERFULL_RUNTIME_CONNSTATE_ACTIVECOUNTER=http.server.connstate.active
       - SERVERFULL_RUNTIME_CONNSTATE_NEWGAUGE=http.server.connstate.new.gauge
       - SERVERFULL_RUNTIME_CONNSTATE_NEWCOUNTER=http.server.connstate.new
-      - SERVERFULL_RUNTIME_LOGGER_OUTPUT=STDOUT
-      - SERVERFULL_RUNTIME_LOGGER_LEVEL=INFO
-      - SERVERFULL_RUNTIME_STATS_OUTPUT=DATADOG
-      - SERVERFULL_RUNTIME_STATS_DATADOG_PACKETSIZE=32768
-      - SERVERFULL_RUNTIME_STATS_DATADOG_TAGS=
-      - SERVERFULL_RUNTIME_STATS_DATADOG_FLUSHINTERVAL=10s
-      - SERVERFULL_RUNTIME_STATS_DATADOG_ADDRESS=statsd:8125
-      - SERVERFULL_RUNTIME_SIGNALS_INSTALLED=OS
-      - SERVERFULL_RUNTIME_SIGNALS_OS_SIGNALS=15 2
-      - IPS_DEVICE42PAGEFETCHER_ENDPOINT=https://ipam-stg.net.atlassian.com/api/1.0/ips
-      - IPS_DEVICE42PAGEFETCHER_LIMIT=500
-      - SUBNETS_DEVICE42PAGEFETCHER_ENDPOINT=https://ipam-stg.net.atlassian.com/api/1.0/subnets
-      - SUBNETS_DEVICE42PAGEFETCHER_LIMIT=500
-      - CUSTOMER_DEVICE42PAGEFETCHER_ENDPOINT=https://ipam-stg.net.atlassian.com/api/1.0/customers
-  gateway:
+      - IPAMFACADE_LAMBDAMODE: "false"
+      - IPAMFACADE_PRODUCER_TYPE: "POST"
+      - IPAMFACADE_PRODUCER_BENTHOS_YAML: ""
+      - IPAMFACADE_PRODUCER_POST_ENDPOINT: "http://benthos-output:8086/publish"
+      - IPAMFACADE_PRODUCER_POST_HTTPCLIENT_TYPE: "DEFAULT"
+      - IPAMFACADE_PRODUCER_POST_HTTPCLIENT_DEFAULTCONFIG_CONTENTTYPE: "application/json"
+      - IPAMFACADE_PRODUCER_POST_HTTPCLIENT_SMART_OPENAPI: ""
+      - IPAMFACADE_DEVICE42CLIENT_ENDPOINT: "https://ipam-stg.net.atlassian.com"
+      - IPAMFACADE_DEVICE42CLIENT_LIMIT=500
```

Comment:
Task: it looks like the `IPAMFACADE_DEVICE42CLIENT_HTTP*` env vars are missing

Reply:
Task: it looks like this still has mixed var syntax

Reply:
info: I believe `- KEY: VALUE` is actually a list of map values and not map values. I believe they all need to have the `-` removed.